blazor/dropdown-list/how-to/blazor-web-assembly-dropdown-list.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: Getting Started With Blazor DropDown List in Web Assembly | Syncfusion
+description: Checkout and learn about getting started with Blazor WebAssembly and Blazor DropDown List component in Visual Studio and much more.
+platform: Blazor
+control: DropDown List
+documentation: ug
+---
+
+# Blazor DropDown List Component in WebAssembly App using Visual Studio
```

Comment:
Why we are adding new page in UG?

Reply:
As per my lead instructed , for the keyword Blazor Web assembly we have to add a separate page in UG already server related page having so we are newly adding for web assembly alone